app/views/events/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h3>Title: <%= @event.title %></h3>
+<p><strong>Event date: </strong><%= @event.event_date %></p>
+<p><strong>Description: </strong><%= @event.description %></p>
+
+<% if @event.attendees.empty? %>
+  <p>No attandees</p>
+<% else %>
+  <strong>Attendees:</strong>
+  <ul>
+    <% @event.attendees.each do |attendee| %>
+      <li><%= link_to attendee.full_name, user_path(attendee.id) %></li>
+    <% end %>
+  </ul>
```

Comment:
## Remove all the logic related code 

* [ ] View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)
* [ ] There shoud be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* Check this article on [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code) and [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more info about this. Thank you. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright.